lib/components_test.go

```diff
@@ -0,0 +1,39 @@
+package lib
```

Comment:
Ideally this would be the `lib_test` package, right? Or is that only when using Ginkgo?